packages/vouching/package.json

```diff
@@ -13,7 +13,9 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "openzeppelin-solidity": "^1.12.0"
+    "openzeppelin-solidity": "^1.12.0",
```

Comment:
I think you don't need this one, please remove if so

Reply:
The package is still using this dependency in the ZepToken contract ([https://github.com/zeppelinos/zos/blob/vouching-upgradeable/packages/vouching/contracts/ZepToken.sol#L3](https://github.com/zeppelinos/zos/blob/vouching-upgradeable/packages/vouching/contracts/ZepToken.sol#L3)). I believe it is better to keep this PR simple, and later remove the dependency once we make ZepToken upgradeable. Do you agree ?

Reply:
sure! let's do that